### DISCUSSION TITLE
"Add to Steam" appears to be resetting user's custom Non-Steam shortcuts and categories

### DISCUSSION THREAD

**Discussion by BobbyWibowo** [neon-nyan/Collapse#397] [Q&A] [answered]
As the title says, the new "Add to Steam" option in Pre-Release seem to have that side effect on my end.

---

**EDIT:** Correction, user's custom categories still exist. It's just custom non-Steam shortcuts all disappeared.

---

Also, I don't know if it's a misconfiguration on my part, but the new `open` launch parameter doesn't appear to be working properly when used via Steam shortcuts anyway. It'll always open Collapse to the last opened game (i.e., if it had Genshin selected before closing Collapse, launching HI3 shortcut from Steam will open Collapse, but it'll stay with Genshin instead of automatically changing to HI3).

But I can confirm that the parameter does work properly when done manually via terminal.

About Steam:
```
Steam Beta Branch:  Stable Client
Steam Version:  1705108172
Steam Client Build Date:  Sat, Jan 13 7:54 AM UTC -08:00
Steam Web Build Date:  Sat, Jan 13 12:02 AM UTC -08:00
Steam API Version:  SteamClient021
```

Collapse Launcher **1.73.2 Preview**

**Comment by BobbyWibowo** [comment]
cc @gablm

**Comment by gablm** [comment, marked as answer]
Hi, thanks for the report!

I cannot reproduce the issue about non-steam shortcuts disappearing, both using stable or beta Steam branches.
Would you be able to describe further how to reproduce it? (if you added various regions, if you added tags to it, etc...)

About the ```open``` issue, that is indeed a bug and was already fixed in CollapseLauncher/Collapse@8265f08 and was shipped today in the 1.73.3 preview release. (you will need to add the region(s) again to Steam for the fix to take effect)

Finally, for the future it is preferred if you use the [issues tab of GitHub](https://github.com/CollapseLauncher/Collapse/issues) for this type of posts.

**Reply by BobbyWibowo** [reply]
> I cannot reproduce the issue about non-steam shortcuts disappearing, both using stable or beta Steam branches.
> Would you be able to describe further how to reproduce it? (if you added various regions, if you added tags to it, etc...)

I just added a few non-steam shortcuts into Steam for other unrelated games/apps. Fully closed Steam. Then used the "Add to Steam" option on Collapse for all 3 games. Opened Steam. The 3 games from Collapse are added properly, but my other non-steam shortcuts that I added previously are now gone. I can confirm the same behavior still exists in 1.73.3 Preview.

Aside from that, I can confirm `open` issue has been fixed 👍

**Reply by gablm** [reply]
After trying again, I had no luck trying to reproduce the problem.
Would you be able to send Collapse's ```.log``` file and your Steam's ```shortcuts.vdf``` file?
 - Collapse logs can be found by going into the launcher's settings, selecting ```Open App Data Folder``` and opening the ```_logs``` folder.
 - The Steam shortcuts file is located at ```(steam folder)\userdata\(steam id)\config\```.
